src/patternfly/demos/Page/examples/Page.md

```diff
@@ -135,4 +137,14 @@ wrapperTag: div
     {{/card}}
   {{/page-main-section}}
 {{/inline}}
-```
\ No newline at end of file
+```
+
+### Context selector in sidebar
+```hbs isFullscreen
+{{> page-template page-template--id="nav-basic-example" page-template-sidebar--hasContextSelector='true'}}
```

Comment:
Our partial params (always I think?) start with a capital letter. And if it's just a boolean, I think we're moving to using `true` and `false` instead of a string for `"true"` 

```suggestion
{{> page-template page-template--id="nav-basic-example" page-template-sidebar--HasContextSelector=true}}
```